test/functorch/test_aotdispatch.py

```diff
@@ -1172,7 +1228,8 @@ def inp_callable():
             c = base1_.view(-1)
             return [base1, base2], [a, b, c]
 
-        fw_graph = self.verify_aot_autograd(f, inp_callable, test_mutation=True)
+        self.verify_aot_autograd(f, partial(inp_callable, req_grad=False), test_mutation=True)
+        fw_graph = self.verify_aot_autograd(f, partial(inp_callable, req_grad=True), test_mutation=True)
```

Comment:
it kind of feels like maybe we should have made these tests parametrized

Reply:
I started to and ended up not. Partially because `verify_aot_autograd` is already annoyingly complicated, and also because a subset of tests are specifically meant to operate on inputs with mixed requires_gradness. But if you'd prefer the refactor, happy to do it!